src/objects/Unit.ts

```diff
@@ -17,6 +17,10 @@ export class Unit extends Struct({
         rangedWoundRoll: UInt32.from(2),
         rangedSaveRoll: UInt32.from(6),
         rangedDamage: UInt32.from(2),
+        meleeHitRoll: UInt32.from(1),
+        meleeWoundRoll: UInt32.from(2),
+        meleeSaveRoll: UInt32.from(6),
```

Comment:
Technically a Unit just has a `saveRoll` which is used when hit by both ranged and melee attacks, so we don't need separate ones here. If you need this for some other reason related to the proof that's fine, but otherwise I'd say it's not needed.

Reply:
Good to know!  The smaller these models are, the better